alot/buffers/search.py

```diff
@@ -61,20 +61,26 @@ def rebuild(self, reverse=False):
         if exclude_tags:
             exclude_tags = [t for t in exclude_tags.split(';') if t]
 
-        try:
-            self.result_count = self.dbman.count_messages(self.querystring)
-            threads = self.dbman.get_threads(
-                self.querystring, order, exclude_tags)
-        except NotmuchError:
-            self.ui.notify('malformed query string: %s' % self.querystring,
-                           'error')
-            self.listbox = urwid.ListBox([])
-            self.body = self.listbox
-            return
-
-        self.threadlist = IterableWalker(threads, ThreadlineWidget,
-                                         dbman=self.dbman,
-                                         reverse=reverse)
+        querylist = self.querystring.split(' / ')
```

Comment:
What happens if one of the queries is empty? Currently the interface stops me if I start a completely empty query. So what happens with this code when I query for `foo / `?

Reply:
In this case the trailing space will be stripped. So you search for `foo /`, the query won't be split and get the same result as without this PR.

When you search for `foo / ''`, nothing will be stripped. Your query will be split and you get the result for `foo` (probably a non-empty list) plus the results for `''` (an empty list). I guess that's what you expect. Do you think, the user should see a warning here? Like `empty subquery string`!?

Reply:
Well if it does not fail dramatically I am fine :)